app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Models\Role;
+use App\Http\Requests\UserAddRequest;
+use App\Http\Requests\UserEditRequest;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+
+class UserController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users = User::all();
+        $searchKeyWord = "";
+
+        return view($this->path_to_view . 'listUser', compact(['users', 'searchKeyWord']));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view($this->path_to_view . 'addUser');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(UserAddRequest $request)
+    {
+        $password = Hash::make($request->password);
+        $user = new User;
+        $user = User::create([
+            'name' => $request->name,
+            'password' => $password,
+            'role_id' => $request->role_id
+        ]);
```

Comment:
```suggestion
        $user = User::create([
            'name' => $request->name,
            'password' => $password,
            'role_id' => $request->role_id,
        ]);
```
code thừa thãi và thiếu dấu ,
chưa fix comment này